packages/ens-utils/src/registrar.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, expect, it } from "vitest";
+import { scaleAnnualPrice } from "./registrar";
+import { buildPrice } from "./price";
+import { Currency } from "./currency";
+import { buildDuration, SECONDS_PER_YEAR } from "./time";
+
+describe("scaleAnnualPrice() function", (t) => {
+  it("should scale a one year price to a half an year price", () => {
+    const price = buildPrice(100n, Currency.Usd);
+    const years = 0.5;
+    const secondsPerYearAsNumber = Number(SECONDS_PER_YEAR.seconds);
```

Comment:
@FrancoAguzzi There's a much easier way to achieve this goal. See `scaleDuration`.